integrated_channels/sap_success_factors/admin/__init__.py

```diff
@@ -32,11 +35,42 @@ class SAPSuccessFactorsEnterpriseCustomerConfigurationAdmin(admin.ModelAdmin):
     """
     Django admin model for SAPSuccessFactorsEnterpriseCustomerConfiguration.
     """
+
     list_display = (
         "enterprise_customer_uuid",
         "sapsf_base_url",
         "key",
+        "secret",
+        "sapsf_company_id",
+        "sapsf_user_id",
+        "sap_access_token",
+    )
+
+    readonly_fields = (
+        "sap_access_token",
     )
 
     class Meta(object):
         model = SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+    def sap_access_token(self, obj):
```

Comment:
Since this is actually a boolean question we're asking of the system, we could rename it to something like 'has_access_token'